### DISCUSSION TITLE
How to post/redirect/get

### DISCUSSION THREAD

**Discussion by ystreibel** [fastapi/fastapi#7904] [Questions] [answered]
### First check

* [X] I used the GitHub search to find a similar issue and didn't find it.
* [X] I searched the FastAPI documentation, with the integrated search.
* [X] I already searched in Google "How to post/redirect/get in FastAPI" and didn't find any information.

### Description

How can I post/redirect/get?

```
@app.post("/")
async def login():
     return RedirectResponse(url="/ressource/1")

@app.get("/ressource/{r_id}")
async def get_ressource(r_id: UUID):
     return {"r_id": r_id}
```

**Comment by ystreibel** [comment, marked as answer]
I found the way to do this. You need to change the redirect default code 307 to 302.

```
...
return RedirectResponse(url="/ressource/1", status_code=HTTP_302_FOUND)
...
```

**Comment by tiangolo** [comment]
Thanks for reporting back and closing the issue :+1:

**Comment by BartlomiejSkwira** [comment]
You can also use `303 (HTTP_303_SEE_OTHER)` which was designed to handle redirects after a POST request. It's also mentioned on wiki for PRG pattern https://en.wikipedia.org/wiki/Post/Redirect/Get
